### PR TITLE
exclude some not tested files. Improve tests or more coverage

### PR DESCRIPTION
**Pull request by ekwanNTT**
# Description

This PR includes the following proposed change(s):

- Configure Jacoco to ignore configuration files
- Improve Unit tests for better code coverage 
- Clean up link files for unnecessary conversion

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

## How Has This Been Tested?

- Ran the unit tests
- Inspected the Jacoco reports

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -319,6 +319,12 @@
 				<groupId>org.jacoco</groupId>
 				<artifactId>jacoco-maven-plugin</artifactId>
 				<version>0.8.7</version>
+				<configuration>
+					<excludes>
+						<exclude>**/Configuration/*</exclude>
```
Comment (ekwanNTT) [inline pom.xml:324]: Config files only not loaded during test

File: pom.xml [hunk 2]
```diff
@@ -319,6 +319,12 @@
 				<groupId>org.jacoco</groupId>
 				<artifactId>jacoco-maven-plugin</artifactId>
 				<version>0.8.7</version>
+				<configuration>
+					<excludes>
+						<exclude>**/Configuration/*</exclude>
+						<exclude>**/ScssApplication.class</exclude>
```
Comment (ekwanNTT) [inline pom.xml:325]: Needlessly lower code coverage as test execution does not load full application

File: src/main/java/ca/bc/gov/open/Scss/Controllers/FileController.java [hunk 3]
```diff
@@ -71,11 +71,7 @@ public LinkFileResponse linkFile(@RequestPayload LinkFile search) {
         UriComponentsBuilder builder =
                 UriComponentsBuilder.fromHttpUrl(host + "LinkFiles")
                         .queryParam("caseActionNumber", search.getCaseActionNumber())
-                        .queryParam(
```
Comment (ekwanNTT) [inline src/main/java/ca/bc/gov/open/Scss/Controllers/FileController.java:74]: Unneeded as Big Decimal will go to a string easily

File: src/main/java/ca/bc/gov/open/Scss/Controllers/HealthController.java [hunk 4]
```diff
@@ -44,7 +44,7 @@ public GetHealthResponse getHealth(@RequestPayload GetHealth empty) {
 
     @PayloadRoot(namespace = SoapConfig.SOAP_NAMESPACE, localPart = "getPing")
     @ResponsePayload
-    public GetPingResponse getHealth(@RequestPayload GetPing empty) {
+    public GetPingResponse getPing(@RequestPayload GetPing empty) {
```
Comment (ekwanNTT) [inline src/main/java/ca/bc/gov/open/Scss/Controllers/HealthController.java:47]: Error from previous commits

File: src/main/java/ca/bc/gov/open/Scss/Models/Serializers/InstantDeserializer.java [hunk 5]
```diff
@@ -22,8 +22,8 @@ public Instant deserialize(JsonParser jsonParser, DeserializationContext deseria
                             .parse(jsonParser.getText());
             return d.toInstant();
         } catch (ParseException e) {
-            log.error(e.getLocalizedMessage());
+            log.error("Could not parse date " + jsonParser.getText());
```
Comment (ekwanNTT) [inline src/main/java/ca/bc/gov/open/Scss/Models/Serializers/InstantDeserializer.java:25]: Better logging as exceptions can display sensitive information.

**Review by HypnoMama — APPROVED** [review]

**Pull request merged**